middle_end/flambda/types/template/flambda_type.templ.ml

```diff
@@ -78,6 +78,16 @@ let unknown_with_subkind kind =
   | Boxed_int64 -> any_boxed_int64 ()
   | Boxed_nativeint -> any_boxed_nativeint ()
   | Tagged_immediate -> any_tagged_immediate ()
+  | Block { tag; fields } ->
+    let fields = List.map unknown_with_descr fields in
+    immutable_block
+      ~is_unique:false (* ? *)
```

Comment:
This needs checking (cc @lthls ).  I imagine in general that any value at Lambda level that is annotated as `Pblock` should not have any uniqueness constraints, in which case this code is correct.  However are we sure we don't end up annotating blocks that do have uniqueness constraints?  I'm not certain.

Reply:
@lpw25 and I have discussed this, and it's ok.